packages/cms-lib/lib/config.js

```diff
@@ -27,10 +27,13 @@ let environmentVariableConfigLoaded = false;
 
 const getConfig = () => _config;
 
-const getConfigFilename = () => {
+const getRelativeConfigFilename = () => {
   const configPathSegments = _configPath.split('/');
 
-  return configPathSegments[configPathSegments.length - 1];
+  return path.relative(
+    process.cwd(),
```

Comment:
This needs to be relative to the ignore file and this code makes the assumption that the path is relative to process.cwd().